deploy/deploy-centralvenv.sh

```diff
@@ -0,0 +1,1000 @@
+    #          at destination /data/tmp/WMCore.venv3/ and using 'Some security string' as a security string for operationss at runtime:
+    # ./deploy-centralvenv.sh -c cmsweb-test1.cern.ch -i test -l wmcore==2.0.3rc1 -d /data/tmp/WMCore.venv3/ -m "Some security string"
+
+    # Example: Same as above, but do not cleanup deployment area and reuse it from previous installtion - usefull for testing behaviour
+    #          of different versions during development or mix running from source and from pypi installed packages.
+    #          NOTE: The 'current' link must point to the proper deployment area e.g. either to 'srv/master' for running from source
+    #                or to 'srv/2.0.3rc1' for running from pypi installed package):
+    # ./deploy-centralvenv.sh -c cmsweb-test1.cern.ch -n -i test -l wmcore==2.0.3rc1 -d /data/tmp/WMCore.venv3/ -m "Some security string"
+
+    # Example: Deploy WMCore central services from source repository, use tag 2.0.0.pre3, linked with 'cmsweb-test1.cern.ch' as a frontend
+    #          at destination /data/tmp/WMCore.venv3/ and using 'Some security string' as a security string for operationss at runtime:
+    # ./deploy-centralvenv.sh -c cmsweb-test1.cern.ch -s -t 2.0.0.pre3 -p "10003 9998" -d /data/tmp/WMCore.venv3/ -m "Some security string"
+
+    # Example: Same as above, but assume 'Yes' to all questions. To be used in order to chose the default flow and rely only
+    #          on the pameters set for configuring the deployment steps. This will avoid human intervention during deployment:
+    # ./deploy-centralvenv.sh -c cmsweb-test1.cern.ch -y -s -t 2.0.0.pre3 -p "10003 9998" -d /data/tmp/WMCore.venv3/ -m "Some security string"
+
+
+    # DEPENDENCIES: All WMCore packages have OS or external libraries/packages dependencies, which are not having a pypi equivalent.
+    #               So far those has been resolved through the set of *.spec files maintained at: https://github.com/cms-sw/cmsdist/tree/comp_gcc630
+    #               Here follows the list of all direct (first level) dependencies per service generated from those spec files:
+
+    #               acdcserver: [python3, rotatelogs, couchdb]
+    #               reqmgr2ms : [python3, jemalloc, rotatelogs, mongo]
+    #               reqmgr2   : [python3, jemalloc, rotatelogs, couchdb]
+    #               reqmon    : [python3, jemalloc, rotatelogs]
+    #               t0_reqmon : [python3, jemalloc, rotatelogs]
+    #               workqueue : [python3, jemalloc, rotatelogs, couchdb, yui]
+
+    #               The above list is generated from the 'cmsdist' repository by:
+    #               git clone https://github.com/cms-sw/cmsdist/tree/comp_gcc630
+    #               python WMCore/bin/adhoc-scripts/ParseSpecCmsswdist.py -a -d cmsdist/ -f <service.spec, e.g. reqmgr2ms.spec>
+
+EOF
+}
+
+usage(){
+    echo -e $1
+    help
+    exit 1
+}
+
+_realPath(){
```

Comment:
Todor I still do not see docstring for this function, may be you missed to commit it.

Reply:
I just added it.